src/libraries/System.Net.Http/src/System/Net/Http/HttpUtilities.cs

```diff
@@ -15,16 +14,13 @@ internal static partial class HttpUtilities
 
         internal static HttpVersionPolicy DefaultVersionPolicy => HttpVersionPolicy.RequestVersionOrLower;
 
-        internal static bool IsHttpUri(Uri uri)
-        {
-            Debug.Assert(uri != null);
-            return IsSupportedScheme(uri.Scheme);
-        }
-
         internal static bool IsSupportedScheme(string scheme) =>
             IsSupportedNonSecureScheme(scheme) ||
             IsSupportedSecureScheme(scheme);
 
+        internal static bool IsSupportedNonSecureScheme(string scheme) =>
```

Comment:
Is this file specific to SocketsHttpHandler now? If so should it be moved there?